### PR TITLE
Get password from user input when not passed in via flag

### PR DESCRIPTION
**Pull request by jere-mie**
When the user doesn't pass in a password using the `-password` flag, we now force the user to enter one.

Password input does not echo back what the user types, making it secure

Also, I added a devcontainer.json so users of devcontainers/GitHub codespaces have the proper version of Go installed by default

### CODE REVIEW

File: cmd/cmd.go [hunk 1]
```diff
@@ -29,6 +32,17 @@ func Run() error {
 		fmt.Printf("hi: %v\n", *quiet)
 	}
 
+	// getting password if it isn't passed in
+	if len(*password) == 0 {
+		fmt.Print("Enter Password: ")
+		bytepw, err := term.ReadPassword(int(syscall.Stdin))
```
Comment (prairir) [inline cmd/cmd.go]:
```suggestion
		bytepw, err := term.ReadPassword(os.Stdin)
```
youd also need to fix the imports

**Review by prairir — CHANGES_REQUESTED** [review]
just make this change but great work

File: cmd/cmd.go (hunk 1 above)
Comment (jere-mie) [reply cmd/cmd.go]: ez fix

Comment (jere-mie) [reply cmd/cmd.go]: ReadPassword() expects an int fd, so the change isn't exactly what you put but I think this should be sufficient

**Review by prairir — APPROVED** [review]

**Pull request merged**